### PR TITLE
Modify Exp functionality

### PR DESCRIPTION
**Pull request by Arrays-Arr**
These modifications will allow unlimited levels and give exp a popup on use for the individual user while not broadcasting or targeting anyone

### CODE REVIEW

File: wavelength-plugins/exp.js [hunk 1]
```diff
@@ -152,13 +140,27 @@ exports.commands = {
...
+					"Level 5 unlocks a free Custom Symbol. <br /><br />" +
+					"Level 10 unlocks a free Custom Avatar. <br /><br />" +
+					"Level 15 unlocks a free Profile Title. <br /><br />" +
+					"Level 20 unlocks a free Custom Userlist Icon. <br /><br />" +
+					"Level 25 unlocks a free Emote. <br /><br />" +
+					"Level 30 unlocks a free Custom Color.  <br /><br />" +
+					"Level 35 unlocks 50 " + currencyPlural + ". <br /><br />" +
+					"Level 40 unlocks a free chatroom. <br /><br />"
+				);
+			});
```
Comment (HoeenCoder) [inline wavelength-plugins/exp.js]: This would be better if it was a reply box rather than a popup.

Comment (Arrays-Arr) [reply wavelength-plugins/exp.js]: well this is for the user only i plan on modifying it later to show what they have unlocked and what they have not unlocked but this is just the basis of that

Comment (Arrays-Arr) [reply wavelength-plugins/exp.js]: but then again it makes no difference if they cant broadcast the bigger reply box

Comment (Arrays-Arr) [reply wavelength-plugins/exp.js]: done

**Pull request merged**